pynhm/utils/parameters.py

```diff
@@ -121,7 +121,7 @@ def _load_prms_parameters(parameter_file):
                         for jj in range(num_vals):
                             line = f.readline().rstrip()
                             line_num += 1
-                            vs[jj] = float(line)
+                            vs[jj] = float(line.split()[0])
```

Comment:
concerning. will it break my tests using parameters?

Reply:
i still think we should have a parameters/ dir. considerable/more time is generally spent on the parameters of this model than on the states, parameters should be first class!